src/cmem/markdown/Markdown.tsx

```diff
@@ -1,12 +1,12 @@
 import React from "react";
 import ReactMarkdown from "react-markdown";
-import { PluggableList } from "react-markdown/lib/react-markdown";
 import { Prism as SyntaxHighlighter } from "react-syntax-highlighter";
 // @ts-ignore: No declaration file for module (TODO: should be @ts-expect-error but GUI elements is used inside project with `noImplicitAny=false`)
 import remarkTypograf from "@mavrin/remark-typograf";
 import rehypeRaw from "rehype-raw";
 import { remarkDefinitionList } from "remark-definition-list";
 import remarkGfm from "remark-gfm";
+import { PluggableList } from "unified";
```

Comment:
Looks like this `PluggableList` has another type definition compared to the old one. At least DI cannot transpile correctly because of of non-assignable types to `reHypePlugins`.

If there is no way to prevent this breaking change we need to plan this update to a next major version, and we need to document it in the changelog.

However, best option would be support both, `PluggableListOldDeprecated` (from react-markdown/lib/react-markdown) and PluggableList (from unified), for `reHypePlugins`, and adding a deprecate note to it regarding `PluggableListOldDeprecated`.